### ISSUE TITLE
Add term temporomandibular joint dislocation

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10457]
**Preferred term label:**
Temporomandibular joint dislocation

**Synonyms**
jaw dislocation, mandibular dislocation

**Definition (free text, please give PubMed ID)**
when the lower jawbone (mandible) pulls away from the temporomandibular joints (TMJ). It is characterized by pain, clicking and functional restriction.
PMID26668447

**Parent term (use hpo.jax.org/app)**
Joint dislocation HP:0001373

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROFACIAL DYSOSTOSIS 1, NAGER TYPE (omim154400), for example (PMID: 23568615)
MARFAN SYNDROME (omim154700), due to laxity of temporomandibular ligaments (PMID: 1508520)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000814
- label: Temporomandibular joint dislocation
- parents: [Abnormality of the temporomandibular joint (HP:0010754)](https://hpo.jax.org/app/browse/term/HP:0010754)
- pmid: PMID:26668447
- def: The lower jawbone (mandible) pulls away from the temporomandibular joints (TMJ). It is characterized by pain, clicking and functional restriction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10457
- synonyms:
